packages/manager/modules/pci/src/error/error.html

```diff
@@ -0,0 +1,41 @@
+<div class="pci-projects-new-error row justify-content-md-center mb-5">
+    <div class="col-md-6">
+
+        <div class="text-center mb-3">
+            <img data-ng-src="{{ :: $ctrl.getAssets($ctrl.error.code) }}"
+                 data-ng-srcset="{{ :: $ctrl.getAssets($ctrl.error.code, true) }}"
+                 alt="" />
+        </div>
+
+        <h1 class="oui-heading_4"
+            data-translate="pci_error_title">
+        </h1>
+
+        <oui-message
+            data-type="error">
+            <span data-translate="{{ 'pci_error_' + $ctrl.error.code }}"
+                  data-translate-default="{{ 'pci_error_default' | translate }}"
```

Comment:
```suggestion
                  data-translate-default="{{:: 'pci_error_default' | translate }}"
```

Reply:
Does not work when using one time binding :-$